ha/test/integration/event_analyzer/test_alert_filter.py

```diff
@@ -27,14 +27,14 @@
     from cortx.utils.conf_store.conf_store import Conf
     from ha import const
     from ha.core.config.config_manager import ConfigManager
-    from ha.core.event_analyzer.filter.filter import AlertFilter
+    from ha.core.event_analyzer.filter.filter import ClusterResourceFilter
     import traceback
     try:
         ConfigManager.init("test_alert_filter")
-        AlertFilter = AlertFilter()
+        AlertFilter = ClusterResourceFilter()
         print("********Alert Filter********")
         resource_type = "enclosure:fru:fan"
```

Comment:
This "resource_type" is not supported. Have you intentionally passed it to fail the test?